sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -225,7 +225,14 @@ def test_confusion_matrix_contrast(pyplot):
     assert_allclose(disp.text_[1, 0].get_color(), max_color)
     assert_allclose(disp.text_[1, 1].get_color(), min_color)
 
+    # Non-regression test for #16442
```

Comment:
Actually I wanted to test `plot_confusion_matrix` but since it takes `estimator` we will never accept the case that we have 0 samples. So you can remove this test.